test/Numerics/DGMethods/compressible_navier_stokes_equations/three_dimensional/ThreeDimensionalCompressibleNavierStokesEquations.jl

```diff
@@ -279,8 +279,8 @@ end
     ρu = state.ρu
     ρθ = state.ρθ
 
-    u = ρu / ρ
-    θ = ρθ / ρ
+    u = ρu
+    θ = ρθ
```

Comment:
@sandreza why did you need to do this?

Reply:
I was playing around with momentum diffusion, (which seems like the right thing to do in this case)

Reply:
@jkozdon any thoughts?